objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -69,6 +69,12 @@
     static final int TEST_IMAGE_HEIGHT = 375;
     static final int RESIZED_IMAGE_WIDTH = 100;
     static final int RESIZED_IMAGE_HEIGHT = 75;
+    
+    // For pre cropping tests
+    static final int TOP_LEFT_X_COORDINATE = 50;
+    static final int TOP_LEFT_Y_COORDINATE = 50;
+    static final int CROPPED_WIDTH = 200;
+    static final int CROPPED_HEIGHT = 150;
```

Comment:
Might want to name these constants with some PRECROP initial naming so it's obvious where they are used.